src/TaskRunner/Commands/InstallCommands.php

```diff
@@ -35,33 +31,20 @@ class InstallCommands extends AbstractCommands implements FilesystemAwareInterfa
    * - Check current status of configuration
    * - Import configuration from datastore into activestore.
    *
-   * @command toolkit:install-clone
+   * @command toolkit:install-clean-dev
```

Comment:
In the dockblock says
```
This will download the database if none local then proceed to dump and sync
```
For install-clean no dump should be used, should be done a installation from scratch and import of configuration.

Reply:
Documentation is totally changed.